docs/setup/independent/install-kubeadm.md

```diff
@@ -119,9 +119,16 @@ For each machine:
   The kubelet is now restarting every few seconds, as it waits in a crashloop for
   kubeadm to tell it what to do.
 
-  Note: Disabling SELinux by running `setenforce 0` is required to allow
-  containers to access the host filesystem, which is required by pod networks for
-  example. You have to do this until SELinux support is improved in the kubelet.
+  Note: 
+  - Disabling SELinux by running `setenforce 0` is required to allow containers to access the host filesystem, which is required by pod networks for example. You have to do this until SELinux support is improved in the kubelet.
+  - Some users on RHEL/CentOS 7 have reported issues with traffic being routed incorrectly due to iptables being bypassed. You should ensure `net.bridge.bridge-nf-call-iptables` is set to 1 in your `sysctl` config, e.g.
+    ``` bash
```

Comment:
Add a newline before the code block so that it renders correctly. See the [preview](https://deploy-preview-4340--kubernetes-io-master-staging.netlify.com/docs/setup/independent/install-kubeadm/#installing-kubelet-and-kubeadm).